assets/sass/_app.scss

```diff
@@ -40,9 +39,13 @@ Tags:
 3.0 Variables
 --------------------------------------------------------------*/
 @import "components/variables";
+@import "components/content";
```

Comment:
Yeah we were talked about this with @mitogh . I'll fix it after design and other things are approved. @jrrrsmith